packages/sourcecred/src/core/credGrainView.test.js

```diff
@@ -499,4 +499,201 @@ describe("core/credGrainView", () => {
       ).toThrow("The graph is missing account");
     });
   });
+
+  describe("validation tests", () => {
+    const credGrainViewNoCred = CredGrainView.fromJSON(
+      JSON.parse(
+        '{"participants":[{"active":true,"identity":{"id":"YVZhbGlkVXVpZEF0TGFzdA", "subtype":"USER",\
```

Comment:
This works. An alternative strategy could've been to initialize a valid data structure in a beforeEach, and then modify it to be invalid for each test case. Parsing from string is kinda beneficial to smush it into fewer lines, but you could also have just initialized the "JSON" as a javascript hash and that would keep Flow typing stricter.

Reply:
Noted. I think my motivation was just to be able to see all the data at once.